wcycle/WCYCLE-8.DATA

```diff
@@ -0,0 +1,347 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+--
+-- Copyright (C) 2018 Equinor
```

Comment:
Old test

Reply:
Please use full name "Equinor ASA"

Reply:
that's what I got from the author... @vkip

Reply:
just copy-paste from my side, feel free to update..

Reply:
done.